### PR TITLE
Add BlueSense v1.1

### PR DESCRIPTION
**Pull request by atika**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/bluesense.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  name 'BlueSense'
+  homepage 'https://apps.inspira.io'
+  license :commercial
+
+  auto_updates true
+  depends_on macos: '>= :yosemite'
+
+  app 'BlueSense.app'
+
+  zap delete: [
```
Comment (vitorgalvao) [inline Casks/bluesense.rb]: `zap delete: '~/Library/Preferences/com.inspira.bluesense.plist'`

File: Casks/bluesense.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+cask 'bluesense' do
+  version '1.1'
+  sha256 'f74c90a7723fb752ddf7da4d1825176638f3c211239426f4cf09935c9a4171a3'
```
Comment (vitorgalvao) [inline Casks/bluesense.rb]: Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/f25b6babcd398abf48e33af3d887b2d00de1d661/Casks/kibako.rb).

File: Casks/bluesense.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+  url 'https://apps.inspira.io/updates/bluesense.zip'
+  name 'BlueSense'
+  homepage 'https://apps.inspira.io'
```
Comment (vitorgalvao) [inline Casks/bluesense.rb]: `https://apps.inspira.io/bluesense/`

**Comment by atika** [comment]
Thank you Vitor I will correct this. `zap delete` is for completely delete all files from the system? Because I have also this folder `~/Library/Application\ Scripts/com.inspira.bluesense` but its contains custom AppleScript scripts made by the user, and I don't want this folder to be deleted by mistake.

**Comment by vitorgalvao** [comment]
> Because I have also this folder `~/Library/Application\ Scripts/com.inspira.bluesense` but its contains custom AppleScript scripts made by the user, and I don't want this folder to be deleted by mistake.

Use `rmdir` instead of `delete`. It will only delete the directory if it is empty.

**Comment by atika** [comment]
OK, it's better this way. I finished editing the cask for bluesense as you have recommended to me.

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**